docusaurus-docs/community/policies/backwards-compatibility.md

```diff
@@ -139,42 +139,72 @@ get community feedback.
 
 #### Removing versions of API endpoints
 
-It is not recommended to remove versions of API endpoints. Removing API
-endpoints, or versions of endpoints, breaks backwards compatibility and should
-only be done under exceptional circumstances such as a security vulnerability.
+Removal of an API endpoint is sometimes necessary. Deprecation notices and removals will
+always be available in [release notes](https://github.com/conda-incubator/conda-store/blob/main/CHANGELOG.md). Further, documentation should be updated to reflect these changes. This should include:
+- version number of the release where this was deprecated
+- provide suggestions for alternatives (if possible)
+- provide justification for the removal (such as a link to the issue
+  or CVE that necessitated the removal).
+
+In order to make these kind of breaking changes responsibly, follow the steps
+outlined below.
+
+##### 1. Deprecate the endpoint
+
+In this stage, conda-store should not be introducing a breaking change. The
+deprecation step communicates to users that the endpoint is marked for removal
+and will removed in a future release.
```

Comment:
We must establish a timeline for deprecation/sunsetting to set expectations accordingly.

> and will removed in a future release.

Since we use CalVer, we should leave _at least_ two/three months before removing any endpoints; otherwise, "a future release" can be anything between a couple of weeks to an indeterminate amount.